docs/space-daemon/getting-started.md

```diff
@@ -0,0 +1,717 @@
+### Installation
+
+#### Download the daemon
+
+First thing is go the [latest release](https://github.com/FleekHQ/space-daemon/releases/latest) of the daemon and download the binary version for your Operating System:
+
+![Binary versions](https://gpuente105-team-bucket.storage.fleek.co/imgs/Screen%20Shot%202020-07-07%20at%2012.26.35.png)
+
+#### Run the daemon
+
+Once you downloaded the dameon binary, open a new terminal window and go to the path where you placed the daemon and change its the access permissions by doing:
```

Comment:
Are these instructions for Mac specifically? If so maybe we could highlight that they are and make a note to follow their operation system guides for program permissions if Linux. Also maybe a note saying Windows is not yet supported by the daemon due to a bug in a dependency.